panel/__init__.py

```diff
@@ -49,6 +49,7 @@
 
 from . import layout  # noqa
 from . import links  # noqa
+from . import observers  # noqa
```

Comment:
Should we rename from `panel.observers` to `panel.observer`?

The first is inline with `widgets`, `panes` etc. The second reads better.

Reply:
obsolete